node.js

```diff
@@ -99,10 +99,14 @@ Node.prototype.findByLabel = function (path) {
   return this.children[path[0]]
 }
 
-Node.prototype.findChild = function (path, method) {
-  var child = this.children[path[0]]
+Node.prototype.findChild = function (path, method, caseSensitive) {
```

Comment:
I think we can pass the `caseSensitive` option when creating a node. Ideally Turbofan would see it's not changing and optimize for that.

Another option might be to do some prototype magic, and have a `NodeCaseInsensitive` constructor.